src/metldata/builtin_transformations/common/assumptions.py

```diff
@@ -0,0 +1,160 @@
+# Copyright 2021 - 2024 Universität Tübingen, DKFZ, EMBL, and Universität zu Köln
+# for the German Human Genome-Phenome Archive (GHGA)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"Assumptions utilized in more than one transform types"
+
+from schemapack.spec.schemapack import SchemaPack
+
+from metldata.builtin_transformations.add_content_properties.path import (
+    resolve_schema_object_path,
+)
+from metldata.builtin_transformations.common.instruction import InstructionProtocol
+from metldata.builtin_transformations.common.path.path import RelationPath
+from metldata.builtin_transformations.common.path.path_elements import (
+    RelationPathElement,
+    RelationPathElementType,
+)
+from metldata.transform.exceptions import ModelAssumptionError, MultiplicityError
+
+
+def assert_path_classes_and_relations_exist(*, model: SchemaPack, path: RelationPath):
```

Comment:
For consistency's sake this should have a return type annotation of `None` or the other functions returning `None` should have no annotation.
I'll leave that up to you, but mark the functions and methods where the return type is missing.